package.json

```diff
@@ -40,7 +40,7 @@
     "clean:obsolete-snapshots": "npm test -- -u",
     "create-props-markdown": "node scripts/props-table-gen/generatePropsTables.js",
     "compile": "npm run compile:clean && npm run compile:build",
-    "compile:build": "lerna ls -a | sed 's/@cerner\\///g' | cut -d ' ' -f1 | xargs -I {} babel packages/{}/src --out-dir packages/{}/lib --copy-files",
+    "compile:build": "lerna run compile --parallel",
```

Comment:
I changed this to the style that we use in terra-toolkit, it saves about a minute of time locally, though uses more cpu. (A fine trade off imo)

Before:
npm run compile:build  93.07s user 12.92s system 103% cpu 1:42.69 total

After:
npm run compile:build  133.44s user 23.11s system 371% cpu 42.142 total

Reply:
It's also more colorful, which everyone can appreciate
![](https://media.giphy.com/media/J65pdOUXTyG1q/giphy.gif?cid=ecf05e47afwmgwdyllz29va1vjmp5yj8wy683qko15n7myvg&rid=giphy.gif&ct=g)

Reply:
We should also do the same for framework and clinical next time.